test/parallel/test-crypto-random.js

```diff
@@ -271,6 +271,16 @@ assert.throws(
   }
 );
 
+assert.rejects(
```

Comment:
seems good, but would expect at least a happy-path test.  I assume that the promise implementations aren't generally tested as thoroughly as the callback implementations given they are usually just wrappers?